src/content/docs/en/welcome.mdx

```diff
@@ -76,6 +76,46 @@ As of now, the only data we collect is a count of how many times a package has b
 
 Fyra Labs (the company behind Ultramarine) does not collect any data that can be tied to an individual without express consent from the user.
 
+### Is Ultramarine semi-rolling?
+
+**No**. Ultramarine and Fedora are not semi-rolling or rolling. Instead, they use a stable release scheme. Updates with breaking changes are rolled out only for every release version. Normal updates usually include bug fixes and some new features, but they should not break backwards compatibility.
+
+However, Ultramarine Linux includes Terra, RPM Fusion and Flathub.
+- Terra is an RPM repository provided by Fyra Labs (the team behind Ultramarine Project), and it mainly uses a rolling release scheme.
+- RPM Fusion is another (very popular) third party repository for Fedora. It uses a stable release scheme.
+- Flathub is a cross-distro Flatpak repository. Updates in this repository has nothing to do with Fedora/Ultramarine's release cycles. Strictly speaking, packages in Flathub are rolling because they are updated once upstream releases a version.
+
+### If some packages are updated mid-release, doesn't that mean Ultramarine is unstable/insecure compared to, e.g. Debian?
+
+**No**. That's an incorrect approach to view if a distribution is secure/stable or not.
+- Packages in Fedora/Ultramarine are updated mid-release, but those updates **only include non-breaking changes**. Your computer has virtually a near-zero chance of breaking mid-release due to an update.
+- Undeniably, Ultramarine and Fedora has much more frequent updates compared to distros like Debian, and to some extent, they might be more unstable. However, Fedora/Ultramarine's release cycles make it so that the operating system is still stable enough for most normal uses. Millions of Fedora devices across the world run fine without breaking, which shows Fedora's high stability. Debian's release cycle is also arguably unnecessarily too slow and can cause more hassles than necessary.
+- Debian does not provide updates that are already available on Fedora for months. Users on Debian might miss out a lot, including new features, bug fixes, security patches, etc. This means Debian could be even more insecure than Fedora/Ultramarine.
+
+### Why Ultramarine (instead of other distros like Ubuntu)
+
+- performance
+  - Fedora (the distro Ultramarine is based on) has one of the best IO performance over other distros like Ubuntu.
+  - BTRFS is the default file system, a relatively high-performant file system with features like snapshots. You can backup your files in less than a second (unlike other file systems like ext4).

```

Comment:
remove "high" from "high-performant" (or swap performant for performance)

maybe swap "like" for "such as" but this one is purely stylistic and up to you